content/200-orm/500-reference/050-prisma-client-reference.mdx

```diff
@@ -407,65 +407,49 @@ const adapter = new PrismaNeon(pool)
 const prisma = new PrismaClient({ adapter })
 ```
 
-### <inlinecode>rejectOnNotFound</inlinecode>
```

Comment:
We have users on older versions than Prisma 5 (and probably also links to this section publicly and in our docs). We should keep it until we have another solution to version our documentation.